<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountWindow</class>
 <widget class="QDialog" name="AccountWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="MLabel" name="LogReg_Button" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text" stdset="0">
    <string>Registration</string>
   </property>
  </widget>
  <widget class="MLabel" name="OK_Button" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text" stdset="0">
    <string>OK</string>
   </property>
  </widget>
  <widget class="MLabel" name="label" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string>Forgote the password?</string>
   </property>
  </widget>
  <widget class="MLineEdit" name="lineEdit" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>271</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
   <property name="frame" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MLineEdit" name="lineEdit_2" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>271</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
   <property name="frame" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MLineEdit" name="lineEdit_3" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>271</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
   <property name="frame" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MLabel</class>
  </customwidget>
  <customwidget>
   <class>MLineEdit</class>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
